### ISSUE TITLE
Multiple works from the same edited volume

### ISSUE THREAD

**Issue by folofjc** [dcpurton/biblatex-sbl#124]
I posted a question on [tex.se](https://tex.stackexchange.com/questions/603880/proper-value-of-mincrossrefs-for-biblatex-sbl) about the bibliography when you have multiple volumes from a multivolume work. I think footnote is correct there, but the bibliography is a bit cumbersome. However, it might be correct when you compare it to multiple works from an edited volume.

In SBLHS2 section 6.2.12, when you have multiple works from the same edited volume, the bibliography entry of each has the entire bibliographic information of the edited volume (which would imply that the bibliography for each volume of a multivolume work should also have the complete bibliography of the multi-volume set). However, 6.2.12 states that in the footnotes, subsequent essays from the same edited work should only contain a short reference to the edited work (see the entry for footnote 9 for Attridge). In the `biblatex-sbl-examples` file, it only shows the first citation. Here is a MWE with multiple works from the same edited volume:

```
\documentclass[letterpaper,12pt]{book}
\usepackage{polyglossia}
\setmainlanguage[variant=us]{english}
\usepackage[english=american]{csquotes} 

\begin{filecontents}[overwrite]{customstyles.dbx}
  \DeclareDatamodelEntrytypes{tdict}
\end{filecontents}

\usepackage[style=sbl,citepages=omit,fullbibrefs=true,sblfootnotes=false,citereset=chapter]{biblatex}

\begin{filecontents}[overwrite]{temp.bib}
@collection{Collection1,
   editor    = {Jones, Philipp},
   title     = {Edited Volume},
   subtitle  = {A Collection of Articles},
   publisher = {Good Publishing Company},
   address   = {London},
   date      = {2009}
}

@incollection{EdwardsColl1,
   crossref = {Collection1},
   author   = {Joe Edwards},
   title    = {The First Essay},
   pages    = {81-99}
}

@incollection{CunninghamColl1,
   crossref = {Collection1},
   author   = {David Cunningham},
   title    = {A Second Essay},
   subtitle = {On Writing Multiple Essays},
   pages    = {100-117}
}

@incollection{SmithColl1,
   crossref   = {Collection1},
   author     = {Paul Smith},
   title      = {The Third and Best Essay},
   pages      = {190-210}
}


\end{filecontents}

\addbibresource{temp.bib}

\usepackage{xparse}

\begin{document}

\null\vfill
One work.\footcite[85]{EdwardsColl1} Second
work.\footcite[101]{CunninghamColl1} Third
work.\footcite[200]{SmithColl1}

\clearpage
\printbibliography
\end{document}
```

![Screenshot 2021-07-12 072943](https://user-images.githubusercontent.com/4741202/125235627-0d170c80-e2e3-11eb-9d5b-fac50c0730fc.png)

![Screenshot 2021-07-12 073002](https://user-images.githubusercontent.com/4741202/125235634-0f796680-e2e3-11eb-951b-27381d2f9f41.png)

I think the Bibliography looks correct, but the footnotes do not. They should not have the full citation of the edited work in each footnote.

**Comment by dcpurton** [comment]
Correct. This isn't supported. I managed to do it on my new code. I'm not sure how easy it will be to back port it though.

**Comment by dcpurton** [comment]
There is a post on the blog somewhere about this BTW.

**Comment by folofjc** [comment]
Okay, I figured you already knew about it. I'll wait and try it out in the new code :)

**Comment by dcpurton** [comment]
Here's the output of my new code from your example. Can you see if it looks correct?

![test](https://user-images.githubusercontent.com/6268044/125274428-1ccb3b00-e34d-11eb-9a38-e264d2a4ca85.png)

**Comment by dcpurton** [comment]
As for your second question, in my new code I seem to have assumed a consistent approach for books:

```
\documentclass[letterpaper,12pt]{article}
\usepackage[style=sbl]{biblatex}

\begin{filecontents}[overwrite]{\jobname.bib}
@mvcommentary{dahood:1965-1970,
  author = {Dahood, Mitchell},
  title = {Psalms},
  volumes = {3},
  series = {Anchor Bible},
  shortseries = {AB},
  number = {16--17A},
  location = {Garden City, NY},
  publisher = {Doubleday},
  date = {1965/1970}
}
@commentary{dahood:1965,
  author = {Dahood, Mitchell},
  title = {Psalms I: 1--50},
  shorttitle = {Psalms I: 1--50},
  volume = {1},
  number = {16},
  date = {1965},
  crossref = {dahood:1965-1970}
}
@commentary{dahood:1968,
  author = {Dahood, Mitchell},
  title = {Psalms II: 51--100},
  shorttitle = {Psalms II: 51--100},
  volume = {2},
  number = {17},
  date = {1968},
  crossref = {dahood:1965-1970}
}
\end{filecontents}
\addbibresource{\jobname.bib}
\begin{document}
\null\vfill
Filler text \autocite[85]{dahood:1965}.

Filler text \autocite[101]{dahood:1968}.
\printbibliography
\end{document}
```

![test](https://user-images.githubusercontent.com/6268044/125277677-e68fba80-e350-11eb-83d4-cff02509b1ab.png)

**Comment by dcpurton** [comment]
I wonder if Dahood shouldn't be repeated in the second citation?

**Comment by folofjc** [comment]
> Here's the output of my new code from your example. Can you see if it looks correct?
> 
> ![test](https://user-images.githubusercontent.com/6268044/125274428-1ccb3b00-e34d-11eb-9a38-e264d2a4ca85.png)

Yep, that looks like what I would expect.

**Comment by folofjc** [comment]
> As for your second question, in my new code I seem to have assumed a consistent approach for books:
> 
> ```
> \documentclass[letterpaper,12pt]{article}
> \usepackage[style=sbl]{biblatex}
> 
> \begin{filecontents}[overwrite]{\jobname.bib}
> @mvcommentary{dahood:1965-1970,
>   author = {Dahood, Mitchell},
>   title = {Psalms},
>   volumes = {3},
>   series = {Anchor Bible},
>   shortseries = {AB},
>   number = {16--17A},
>   location = {Garden City, NY},
>   publisher = {Doubleday},
>   date = {1965/1970}
> }
> @commentary{dahood:1965,
>   author = {Dahood, Mitchell},
>   title = {Psalms I: 1--50},
>   shorttitle = {Psalms I: 1--50},
>   volume = {1},
>   number = {16},
>   date = {1965},
>   crossref = {dahood:1965-1970}
> }
> @commentary{dahood:1968,
>   author = {Dahood, Mitchell},
>   title = {Psalms II: 51--100},
>   shorttitle = {Psalms II: 51--100},
>   volume = {2},
>   number = {17},
>   date = {1968},
>   crossref = {dahood:1965-1970}
> }
> \end{filecontents}
> \addbibresource{\jobname.bib}
> \begin{document}
> \null\vfill
> Filler text \autocite[85]{dahood:1965}.
> 
> Filler text \autocite[101]{dahood:1968}.
> \printbibliography
> \end{document}
> ```
> 
> ![test](https://user-images.githubusercontent.com/6268044/125277677-e68fba80-e350-11eb-83d4-cff02509b1ab.png)

This is also what I would expect. Consistency I think is the right approach.

**Comment by folofjc** [comment]
> I wonder if Dahood shouldn't be repeated in the second citation?

I think it should, because for edited volumes the editor is repeated when citing more essays from the same edited work. So I think that the editor of the multivolume work should also be repeated. This is an odd example because the editor of the `mvbook` and the author of the `book` are the same person. But in the question I posted on tex.se, they are different, so it would make sense to repeat them.
